aidial_adapter_bedrock/dial_api/response.py

```diff
@@ -16,13 +18,33 @@ class ModelsResponse(BaseModel):
     data: List[ModelObject]
 
 
+def _encode_vector(
+    encoding_format: Literal["float", "base64"],
+    vector: List[float],
+) -> List[float] | str:
+    base64_encoding = encoding_format == "base64"
```

Comment:
Minor: since it's boolean, I think `is_base64_encoding` is more appropriate

Or just skip this line, and make this check without intermediate variable
```python
def _encode_vector(
    encoding_format: Literal["float", "base64"],
    vector: List[float],
) -> List[float] | str:
    return (
        vector_to_base64(vector) 
        if encoding_format == "base64" 
        else vector
    )
```

Reply:
fixed